builtin_metadata.json

```diff
@@ -10352,6 +10353,36 @@
     },
     "wasm": true
   },
+  "json.marshal_indent": {
+    "args": [
+      {
+        "description": "the term to serialize",
+        "name": "x",
+        "type": "any"
+      },
+      {
+        "description": "the string to indent with, defaults to `\\t` (horizontal tab character, 0x09) if null",
+        "name": "indent",
+        "type": "any\u003cnull, string\u003e"
+      },
+      {
+        "description": "the string to prefix lines with, defaults to empty-string if null",
+        "name": "prefix",
+        "type": "any\u003cnull, string\u003e"
+      }
+    ],
+    "available": [
+      "edge"
+    ],
+    "description": "Serializes the input term to indented JSON, including line breaks, with optional prefix. One or more copies of the indent string are emitted before each child JSON element, depending on indent level.",
+    "introduced": "edge",
+    "result": {
+      "description": "the JSON string representation of `x`, with configured prefix/indent string(s) as appropriate",
+      "name": "y",
+      "type": "string"
+    },
+    "wasm": false
```

Comment:
I'm noticing `make generate` is setting `wasm` to `false`. I'm not sure if this is a bug in my local dev env (I think all of the WASM stuff is being skipped because I don't have Docker installed), but I see that all of the other encoding functions are tagged with `"wasm": true`.

Reply:
Thanks @srenatus for the clarification! It seems like this is intentional as WASM support requires implementing the builtin in C. The JSON serializer in json.c looks like it'd require a significant amount of work to support functionality akin to Go's `MarshalIndent`, so I'd expect this would need to be added separately.